### PR TITLE
chore: changelog

### PR DESCRIPTION
**Pull request by okwme**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/main/README.md#merging-a-pr))

### CODE REVIEW

**Review by yaruwangway — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -373,6 +378,7 @@ See the [Tendermint v0.34.7 SDK changelog](https://github.com/tendermint/tenderm
 
 [Unreleased]: https://github.com/cosmos/gaia/compare/v6.0.0...HEAD
```
Comment (yaruwangway) [inline CHANGELOG.md]: just confirm: the [unrelased] should always link to a relase, not a rc-release. so it should be `[Unreleased]: https://github.com/cosmos/gaia/compare/v6.0.0...HEAD` even though there is no `v6.0.0` yet, we only have `v6.0.0-rc1`

File: CHANGELOG.md [hunk 2]
```diff
@@ -43,9 +43,14 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+## [5.0.7] - 2021-09-30
+
+  * (gaia) Bump Cosmos SDK to 0.42.10
+
 ## [5.0.6] - 2021-09-16
 
  * (gaia) Bump tendermint to 0.34.13
+
  
 ## [5.0.5] - 2021-08-05
```
Comment (yaruwangway) [inline CHANGELOG.md]: from `[5.0.3] - 2021-07-30` to `[6.0.0] - 2021-09-17` we do not link the version to release in change log, which is different from [v5.0.2] - 2021-07-15 and even earlier version.

File: CHANGELOG.md (hunk 1 above)
Comment (okwme) [reply CHANGELOG.md]: could you check with @marbar3778 to confirm what it should look like for a RC?

Comment (tac0turtle) [inline CHANGELOG.md]:
```suggestion
[Unreleased]: https://github.com/cosmos/gaia/compare/v6.0.0-rc1...HEAD
```
sorry couldnt do this in the other comment for some reason. This works

Comment (tac0turtle) [reply CHANGELOG.md]: leaving it to v6.0.0 is also fine. Its one less thing to think of when the release is made. either way works.

**Pull request merged**